util/topgen.py

```diff
@@ -1197,9 +1200,9 @@ def insert_ip_attrs(module: ConfigT, params: ParamsT):
                         _get_alert_handler_params(topcfg))
     # Add rv_plic
     amend_interrupt(topcfg, name_to_block, allow_missing_blocks=True)
-    if "rv_plic" in ipgen_instances:
-        insert_ip_attrs(ipgen_instances["rv_plic"][0],
-                        _get_rv_plic_params(topcfg))
+    for inst in ipgen_instances["rv_plic"]:
```

Comment:
I think you want `ipgen_instances.get("rv_plic", [])` here, to cope with the case that the key isn't there.